src/components/dashboard/metric-link.vue

```diff
@@ -1,20 +1,48 @@
 <script>
 export default {
   name: 'metric-link',
-
+  components: {
+    Widget: () => import('../common/widget.vue')
+  },
   props: {
     amount: String,
     link: String,
-    title: String
+    title: String,
+    icon: String,
+    linkLabel: String
+  },
+  methods: {
+    redirectToLink () {
+      this.$router.push({ path: `${this.link}` })
+      // this.$router.push({ path: `preview/${this.link}` })
+    }
   }
 }
 </script>
 
 <template lang="pug">
-q-btn.fit(:to="{ name: link }" rounded unelevated color="white" size="0.9em")
-  .column.q-py-md
-    .text-h3.text-bold.text-grey-9.text-left.q-pa-md {{ amount }}
-    .row.justify-between.items-center
-      .text-subtitle2.text-no-wrap.text-grey-7 {{ title }}
-      q-icon.q-pl-xl(color="grey-5" size="xs" name="fas fa-chevron-right")
+widget.cursor-pointer
+  #container(@click="redirectToLink")
+    .row
+      .col-6.text-left
+        .icon-container.bg-primary.q-my-sm
```

Comment:
You can do a circle with an icon in it with a single quasar component like so (this avoids custom css and reduces overall amount of code):
`q-btn(round unelevated :icon="icon" color="primary" text-color="white" size="sm" :ripple="false")`